src/components/Data/Documents/Page.vue

```diff
@@ -40,14 +40,25 @@
         <template v-else>
           <b-row class="justify-content-md-center" no-gutters>
             <b-col cols="12">
-              <template v-if="isCollectionEmpty">
+              <template v-if="isCollectionEmpty && !fetchingDocuments">
                 <realtime-only-empty-state
                   v-if="isRealtimeCollection"
                   :index="index"
                   :collection="collection"
                 />
                 <empty-state v-else :index="index" :collection="collection" />
               </template>
+              <template v-if="fetchingDocuments">
```

Comment:
Nitpicking
```suggestion
              <template functional v-if="fetchingDocuments">
```
See: https://learn-vuejs.github.io/vue-patterns/patterns/#functional-component